oamapps/columnstoreSupport/columnstoreSupport.cpp

```diff
@@ -333,7 +332,7 @@ int main(int argc, char* argv[])
   }
 
   // get Local Module Name and Server Install Indicator
-  string singleServerInstall;
+  string singleServerInstall = "y";
```

Comment:
This is in the support report tool, which is out dated. For correctness and since the tool sets it to "y" if it finds one dbrm worker in the xml, it should default to "n". This is corrected.